### ISSUE TITLE
Numerous MCR problems with configuration

### ISSUE THREAD

**Issue by zevelijah** [Social-Evolution-and-Behavior/anTraX#48]
I keep getting this approximately the same series of errors when I try to configure:

```
$ antrax configure anTraX-data/JS16/

==================================================================================

Welcome to anTraX - a software for tracking color tagged ants (and other insects)

==================================================================================

10:38:21 -I- Reading video information from file
MathWorks::System::IUserException [Error using matlab.internal.webwindow (line 333)
MATLABWindow application failed to launch. Unable to launch the MATLABWindow application
Error in matlab.ui.internal.controller.platformhost.CEFFigurePlatformHost/createView (line 58)

Error in matlab.ui.internal.controller.FigureController/createView (line 257)

Error in matlab.ui.internal.componentframework.WebComponentController/add (line 75)

Error in matlab.ui.internal.controller.WebCanvasContainerController/add (line 51)

Error in matlab.ui.internal.componentframework.WebControllerFactory/create (line 94)

Error in matlab.ui.internal.dialog.DialogHelper.validateUIfigure (line 20)

Error in matlab.ui.dialog.ProgressDialog (line 45)

Error in uiprogressdlg (line 33)

Error in antrax/set_experiment (line 315)

Error in antrax/startupFcn (line 1184)

Error in antrax>@(app)startupFcn(app,varargin{:})

Error in appdesigner.internal.service.AppManagementService/tryCallback (line 336)

Error in matlab.apps.AppBase/runStartupFcn (line 41)

Error in antrax (line 3567)

Error in antrax_mcr_interface (line 8)]
MathWorks::System::IUserException [Dot indexing is not supported for variables of this type.
Error in matlab.ui.internal.componentframework.services.core.eventhandling.WebEventHandlingService/setProperty (line 201)

Error in matlab.ui.internal.controller.WebCanvasContainerController/updateSceneChannel (line 138)

Error in matlab.ui.internal.controller.WebCanvasContainerController/set.Canvas (line 90)

Error in matlab.ui.internal.componentframework.WebControllerFactory/create (line 94)

Error in matlab.ui.internal.dialog.DialogHelper.validateUIfigure (line 20)

Error in matlab.ui.dialog.ProgressDialog (line 45)

Error in uiprogressdlg (line 33)

Error in antrax/set_experiment (line 315)

Error in antrax/startupFcn (line 1184)

Error in antrax>@(app)startupFcn(app,varargin{:})

Error in appdesigner.internal.service.AppManagementService/tryCallback (line 336)

Error in matlab.apps.AppBase/runStartupFcn (line 41)

Error in antrax (line 3567)

Error in antrax_mcr_interface (line 8)]
10:38:33 -I- opening video file anTraX-data/JS16/videos//1_6/JS16_1.mp4
Index exceeds the number of array elements (1).
Error in ffinfo (line 30)

Error in ffreader/collectInfo (line 176)

Error in ffreader (line 59)

Error in expreader/open_vid (line 393)

Error in expreader/fetch_frame (line 338)

Error in expreader/read_frame (line 263)

Error in trhandles/read_frame (line 415)

Error in antrax/set_Trck (line 410)

Error in antrax/set_experiment (line 344)

Error in antrax/startupFcn (line 1184)

Error in appdesigner.internal.service.AppManagementService/tryCallback (line 336)

Error in matlab.apps.AppBase/runStartupFcn (line 41)

Error in antrax (line 3567)

Error in antrax_mcr_interface (line 8)
MATLAB:badsubscript

```

Here are my environment variables for both my ~/.profile and ~/.bashrc:
export ANTRAX_MCR=/usr/local/MATLAB/MATLAB_Runtime/R2019a/v96/ # I have a custom location because I couldn't figure out how to fix some installation problems and tried anything. I eventually fixed them but this exists as a waste product.
export ANTRAX_PATH=~/code/anTraX-zev-local/anTraX 
export ANTRAX_USE_MCR=True 
export LD_LIBRARY_PATH=/usr/local/MATLAB/MATLAB_Runtime/R2019a/v96/runtime/glnxa64:/usr/local/MATLAB/MATLAB_Runtime/R2019a/v96/bin/glnxa64:/usr/local/MATLAB/MATLAB_Runtime/R2019a/v96/sys/os/glnxa64:/usr/local/MATLAB/MATLAB_Runtime/R2019a/v96/extern/bin/glnxa64

I am using Ubuntu 24.04 Noble Numbat. I cannot figure out how to even determine what the problem is. I would appreciate your assistance.

**Comment by zevelijah** [comment]
I don't know why that's crossed out. Let me try again:
Here are my environment variables for both my ~/.profile and ~/.bashrc:
export ANTRAX_MCR=/usr/local/MATLAB/MATLAB_Runtime/R2019a/v96/ # I have a custom location because I couldn't figure out how to fix some installation problems and tried anything. I eventually fixed them but this exists as a waste product.
export ANTRAX_PATH=/code/anTraX-zev-local/anTraX
export ANTRAX_USE_MCR=True
export LD_LIBRARY_PATH=/usr/local/MATLAB/MATLAB_Runtime/R2019a/v96/runtime/glnxa64:/usr/local/MATLAB/MATLAB_Runtime/R2019a/v96/bin/glnxa64:/usr/local/MATLAB/MATLAB_Runtime/R2019a/v96/sys/os/glnxa64:/usr/local/MATLAB/MATLAB_Runtime/R2019a/v96/extern/bin/glnxa64

**Comment by yohann-chemtob** [comment]
Hi @zevelijah 
We never try to install antrax on Ubuntu 24.04 but I don't see why that would be an issue.
If you encountered issue when installing Matlab Runtime, I would suggest reinstall it fresh to see if that fix the issue as a first try.
Let me know if the problem persist.

**Comment by zevelijah** [comment]
I am writing this on a borrowed device, because I can no longer log in to my computer. If I can fix it, I will do it all again and get back to you.
